docs/tutorials/one_click_private_tangle.md

```diff
@@ -233,18 +244,24 @@ chmod +x ./private-hornet.sh
 ./private-hornet.sh install "my-node:14266:15601:8082"
 ```
 
-The main parameter is a Node connection string. Such string has different fields separated by a colon (`:`). The first field is the (container and host) name of your Node and, at installation time, it can be followed, optionally, by the TCP port numbers corresponding to the API endpoint, the peering endpoint and the dashboard endpoint.
+The main parameter is a Node connection string. Such string has different fields separated by a colon (`:`). The first field is the (container and host) name of your Node and, at installation time, it can be followed, optionally, by the TCP port numbers corresponding to the API endpoint, the peering endpoint and the dashboard endpoint. 
+
+If no port numbers are provided, i.e. only the container name is supplied, no ports will be exposed to the host. In addition, you can omit some of the ports, but the separator `:` has to be kept, for instance, if you just only want to  expose the dashboard port to the host you can run:
+
+```console
+./private-hornet.sh install "my-node:::8082"
+```
 
-After executing the commands described above a new Docker container (named `my-node`) executing a Hornet node will be running. Such Hornet node will be automatically peered with the `node1` Hornet node. The snapshot and Coordinator's public keys will be taken, respectively, from the configuration and snapshot folders of your Private Tangle. 
+After executing the commands described above a new Docker container (named `my-node`) executing a Hornet node will be running. Such Hornet node will be automatically peered. The snapshot, Coordinator's public keys and autopeering entry node address will be taken, from the snapshot and configuration folders of your Tangle. 
 
-Note: In case you want to spin a node from a different machine (or base folder) you would need to pass manually those parameters including the [multiaddr](https://wiki.iota.org/hornet/post_installation/peering#addressing-peer-neighbors) peer address of the node (for instance `node1`) you want to peer with, as explained [here](https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net/extra-nodes). 
+Note: In case you want to spin a node from a different machine (or base folder) you would need to pass manually those parameters including a [multiaddr](https://hornet.docs.iota.org/post_installation/peering/#addressing-peer-neighbors) peer address of a node to peer with (for instance `node1`) you want to peer with, as explained [here](https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net/extra-nodes). 
```

Comment:
```suggestion
Note: In case you want to spin a node from a different machine (or base folder) you would need to manually pass those parameters including a [multiaddr](https://hornet.docs.iota.org/post_installation/peering/#addressing-peer-neighbors) peer address of a node to peer with (for instance `node1`) you want to peer with, as explained [here](https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net/extra-nodes). 
```